packages/legacy/core/__tests__/screens/__snapshots__/PINEnter.test.tsx.snap

```diff
@@ -296,7 +298,7 @@ exports[`PINEnter Screen PIN Enter renders correctly 1`] = `
                           "top": 0,
                         }
                       }
-                      testID="com.ariesbifold:id/EnterPIN"
+                      testID="com.ariesbifold:id/PINEnter.EnterPIN"
```

Comment:
i18n entries should not be used for testIDs (this testID should remain the same)